### ISSUE TITLE
BUG: SubscribeToPositionUpdatesAsync - incorrect enum

### ISSUE THREAD

**Issue by forReason** [burakoner/OKX.Api#67]
I have an issue with SubscribeToPositionUpdatesAsync():
```
Wrong URL or channel:positions,instType:0 doesn't exist. Please use the correct URL, channel and parameters referring to API document.
```

my best guess is that the instrument type enum is invalidly parsed as int rather than as string. In the official documentation, they seem to use the string in all capital for several Operations:

```
{
    "code": "0",
    "data": [
        {
            "cTime": "1654177169995",
            "ccy": "BTC",
            "closeAvgPx": "29786.5999999789081085",
            "closeTotalPos": "1",
            "instId": "BTC-USD-SWAP",
            "instType": "SWAP",
            "lever": "10.0",
            "mgnMode": "cross",
            "openAvgPx": "29783.8999999995535393",
            "openMaxPos": "1",
            "realizedPnl": "0.001",
            "fee": "-0.0001",
            "fundingFee": "0",
            "liqPenalty": "0",
            "pnl": "0.0011",
            "pnlRatio": "0.000906447858888",
            "posId": "452587086133239818",
            "posSide": "long",
            "direction": "long",
            "triggerPx": "",
            "type": "1",
            "uTime": "1654177174419",
            "uly": "BTC-USD"
        }
    ],
    "msg": ""
}
```

**Comment by forReason** [comment]
I have added a temporary workaround in my fork:
https://github.com/forReason/OKX.Api/blob/master/OKX.Api/Common/Enums/OkxInstrumentType.cs
the fork is not how it should be implemented.. It seems you want to use the [Map] attribute for ApiSharp?

Unfortunately, I dont exactly know what the issue is and I would expect the issue to pop up on other ends as well.
I cant judge yet, since the Project is just starting.
For my quick and Dirty solution is a Custom json converter, which converts the Enum Value to upper case and it works.
Thats unfortunately all I can do with my limited knowledge. I love your Project.

**Comment by burakoner** [comment]
I'll fix this issue tonight.

**Comment by burakoner** [comment]
Fixed in v5.4.1025 with ApiSharp Update. Thanks for feedback.
